packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -108,11 +130,11 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
         </div>
       )}
       <div css={containerStyles}>
-        <div css={buttonsContainer}>
-          <BackLink href={backHref} />
```

Comment:
Removing the BackLink is intentional?

<img width="1433" alt="Screenshot 2023-05-17 at 07 26 24" src="https://github.com/yldio/asap-hub/assets/16595804/5b9fb531-2935-4168-ad1b-f4c22c0b0763">

Reply:
Yes, thats where the duplicate buttons go. I had this confirmed Jao as they were omitted from the designs